UI/MauiEmbedding/ArcGisApp/ArcGisApp/ArcGisApp.csproj

```diff
@@ -1,85 +1,46 @@
-<Project Sdk="Microsoft.NET.Sdk">
-	<PropertyGroup>
-		<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('windows')) or '$(EnableWindowsTargeting)' == 'true'">$(TargetFrameworks);net8.0-windows10.0.19041</TargetFrameworks>
-		<TargetFrameworks>$(TargetFrameworks);net8.0;net8.0-ios;net8.0-android;net8.0-maccatalyst</TargetFrameworks>
-		<TargetFrameworks Condition="'$(OverrideTargetFrameworks)'!=''">$(OverrideTargetFrameworks)</TargetFrameworks>
-
-		<!-- Ensures the .xr.xml files are generated in a proper layout folder -->
-		<GenerateLibraryLayout>true</GenerateLibraryLayout>
-	</PropertyGroup>
-
-	<ItemGroup>
-		<PackageReference Include="Uno.WinUI" />
-		<PackageReference Include="Uno.Resizetizer" />
-		<PackageReference Include="CommunityToolkit.Mvvm" />
-		<PackageReference Include="Uno.Extensions.Configuration" />
-		<PackageReference Include="Uno.Extensions.Logging.WinUI" />
-		<PackageReference Include="Uno.Extensions.Logging.Serilog" />
-		<PackageReference Include="Uno.Extensions.Maui.WinUI" />
-		
-		<PackageReference Include="Uno.Toolkit.WinUI" />
-		<PackageReference Include="Uno.Extensions.Hosting.WinUI" />
-		<PackageReference Include="Uno.Extensions.Localization.WinUI" />
-		<PackageReference Include="Uno.Extensions.Navigation.Toolkit.WinUI" />
-		<PackageReference Include="Uno.Extensions.Navigation.WinUI" />
-		<PackageReference Include="Microsoft.Extensions.Logging.Console" />
-	
-	    <PackageReference Include="Microsoft.Maui.Controls" />
-	    <PackageReference Include="Microsoft.Maui.Controls.Compatibility" />
-		<PackageReference Include="Microsoft.Maui.Graphics" />
+<Project Sdk="Uno.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>
+      net8.0-android;
+      net8.0-ios;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.19041;
+    </TargetFrameworks>
```

Comment:
I suppose it would be better as single line because of https://github.com/unoplatform/uno.templates/pull/684.

Reply:
Let wait for 5.3 release to update this please